thoth/storages/data/alembic/versions/f6d46e0b37da_flag_for_if_specific_hash_has_gone_.py

```diff
@@ -0,0 +1,28 @@
+"""Flag for if specific hash has gone missing, defaults to present
+
+Revision ID: f6d46e0b37da
+Revises: 74254b596f00
+Create Date: 2020-03-24 21:32:57.426880+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f6d46e0b37da'
+down_revision = '74254b596f00'
```

Comment:
Please update the revision references in the file, as another PR that changes the schema has been merged already: https://github.com/thoth-station/storages/pull/1668/files

therefore this change in schema should down revision: `0ea53ec16d6b` (https://github.com/pacospace/storages/blob/16d6929a76bdbf1961ac1d39c768bad445aef4a4/thoth/storages/data/alembic/versions/0ea53ec16d6b_use_text_instead_of_strings_all_over_.py#L3)

Reply:
I did look at upsert.  I still think update is what we want because we give the flag a default value so we should never be inserting.